SofaImGui/src/SofaImGui/windows/Settings.cpp

```diff
@@ -63,25 +63,30 @@ namespace windows
                     ImGui::EndCombo();
                 }
 
+                float globalScale = static_cast<float>(ini.GetDoubleValue("Visualization", "globalScale", 1.0));
+                {
+                    ImGuiIO& io = ImGui::GetIO();
+                    io.FontGlobalScale = globalScale;
+                    constexpr float MIN_SCALE = 0.3f;
+                    constexpr float MAX_SCALE = 2.0f;
+                    ImGui::DragFloat("global scale", &io.FontGlobalScale, 0.005f, MIN_SCALE, MAX_SCALE, "%.2f", ImGuiSliderFlags_AlwaysClamp); // Scale everything
 
-                ImGuiIO& io = ImGui::GetIO();
-                const float MIN_SCALE = 0.3f;
-                const float MAX_SCALE = 2.0f;
-                ImGui::DragFloat("global scale", &io.FontGlobalScale, 0.005f, MIN_SCALE, MAX_SCALE, "%.2f", ImGuiSliderFlags_AlwaysClamp); // Scale everything
-
+                    ini.SetDoubleValue("Visualization", "globalScale", static_cast<double>(io.FontGlobalScale));
+                    [[maybe_unused]] SI_Error rc = ini.SaveFile(sofaimgui::AppIniFile::getAppIniFile().c_str());
```

Comment:
```suggestion
                    if (std::abs(globalScale - io.FontGlobalScale) > 0.005f))
                    {
                        [[maybe_unused]] SI_Error rc = ini.SaveFile(sofaimgui::AppIniFile::getAppIniFile().c_str());
                    }
```
I suggest to save only when a change occurred, otherwise it will be at each frame.

Reply:
Tested and approved